ac/ac-quiz/src/index.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import Form from "react-jsonschema-form";
+
+export const meta = {
+  name: 'Quiz with Multiple-Choice Questions',
+  type: 'react-component'
+}
+
+export const config = {
+  title: 'Configuration for MCQ',
+  type: 'object',
+  properties: {
+    'name': {
+      type: 'string',
+      title: 'Activity name'
+    },
+    'MCQ': {
+      title: "MCQ",
+      type: "array",
+      items: {
+        type: "object",
+        title: "New Question",
+        required: [
+          "title"
+        ],
+        properties: {
+          title: {
+            type: "string",
+            title: "Question"
+          },
+          answers: {
+            type: "array",
+            title: "Possible answers",
+            items: {
+              type: "object",
+              required: [
+                "title"
+              ],
+              properties: {
+                title: {
+                  type: "string",
+                  title: "Answer"
+                },
+                answer: {
+                  type: "boolean",
+                  title: "This is an answer",
+                  default: false
+                }
+              }
+            }
+          },
+          details: {
+            type: "string",
+            title: "Enter an explanation",
+          }
+        }
+      }
+    }
+  }
+}
+
+export const ActivityRunner = ({ config, logger, onCompletion }) => {
+  const propdef = config.MCQ.reduce(
+    (acc, x, i) => ({...acc, [i + '']: { 
```

Comment:
A bit difficult to understand if you use `x`, `i`, `y`, and `j` to name your variables. Couldn't it be `question`, `questionIndex`, `answer` and `answerIndex`?